src/tools/compiletest/src/header/tests.rs

```diff
@@ -284,6 +284,15 @@ fn llvm_version() {
 
     let config: Config = cfg().llvm_version("10.0.0").build();
     assert!(!check_ignore(&config, "//@ min-llvm-version: 9.0"));
+
+    let config: Config = cfg().llvm_version("10.0.0").build();
+    assert!(check_ignore(&config, "//@ exact-llvm-major-version: 9.0"));
```

Comment:
Suggestion: can you add two more test cases:

1. `//@ exact-llvm-major-version: 9` (no minor version component)
2. `//@ exact-llvm-major-version: 10` vs `cfg().llvm_version("10.6.2")`